lang/en/docs/software-directory/scripting/jupyter-lab/data.md

```diff
@@ -0,0 +1,11 @@
+# Jupyter Lab: Structured Representation
```

Comment:
We need to add a note about accessing the ipynb files and the rest of the files:

```
## Files/storage convention

1. Initially, the root of the Dropbox folder is passed to the application on the start, so the files at the root of the Dropbox folder (link) can be accessed
2. Upon each "Save and Checkpoint" action invoked inside the notebook, the ipynb file is overwritten. A new version is stored in the file system, and a checkpoint is saved to the job inside its directory (link).
3. All notebooks have access to the filesystem accessible to the user on the corresponding computational node, namely the home (link) and share (share) directories. For example, the following command will list the shared directory for the account "exabyte-io", when invoked inside the Jupyter Notebook running on "cluster-007": `<COMMAND HERE>`
```

Reply:
Added to Jupyter lab application overview page.